### PR TITLE
St/read write access

### PR DESCRIPTION
**Pull request by showduhtung**

### CODE REVIEW

File: src/components/Main/Schedule/Table.tsx [hunk 1]
```diff
@@ -42,6 +48,7 @@ export const Table = (props: TableProps) => {
           <span className={classes.titleText}>{title}</span>
         </h3>
       )}
+      {access}
```
Comment (hchiam) [inline src/components/Main/Schedule/Table.tsx:51]: this looks like a debug that should eventually be deleted

Comment (hchiam) [inline src/components/Main/Schedule/Table.tsx:51]: this looks like a debug that should eventually be deleted

Comment (showduhtung) [reply src/components/Main/Schedule/Table.tsx:51]: oh no it's ok. this is for proof of concept that user has read or write access

**Review by hchiam — COMMENTED** [review]
LGTM! :+1:

**Pull request merged**